app/services/campaign_page_builder.rb

```diff
@@ -26,9 +26,15 @@ def page
   end
 
   def create_plugins
-    page.liquid_layout.partial_refs.each do |partial, ref|
-      plugin_name = LiquidPartial.find_by(title: partial).plugin_name
-      Plugins.create_for_page(plugin_name, page, ref)
+    if default
+      Plugins.registered.each do |plugin|
+        Plugins.basic_create_for_page(plugin, page)
+      end
```

Comment:
It still feels like a dev-mode hack, so can you change it so that it defaults to false, ie it only uses the dev-mode behavior if you explicitly set the environment variable?